### PR TITLE
Rework tutorial page

### PR DESCRIPTION
**Pull request by kolszewska**
Part of : https://github.com/medtagger/MedTagger/issues/126

## Proposed Changes

 - Reworked tutorial page entirely,
 - Instead of severals gifs, one YT video is available (for now it is a placeholder),
 - **Tutorial now appears AFTER user logs in!**,
 - Updated e2e tests,
- Changed names from "marker-tutorial" to "tutorial"

## UI changes (optional)

tutorial-page

### CODE REVIEW

File: frontend/src/app/pages/login-page/login-page.component.ts [hunk 1]
```diff
@@ -78,7 +81,8 @@ export class LoginPageComponent implements OnInit {
                 console.log(userInfo);
                 if (userInfo) {
                     sessionStorage.setItem('userInfo', JSON.stringify(userInfo));
-                    this.routerService.navigate([appRoutes.HOME]);
+                    this.user = JSON.parse(sessionStorage.getItem('userInfo'));
```
Comment (jpowie01) [inline frontend/src/app/pages/login-page/login-page.component.ts]: Can we do it once during Component initialization?

File: frontend/src/app/pages/tutorial-page/tutorial-page.component.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<app-nav-bar>
+</app-nav-bar>
+
+<div class="container">
+    <div class="tutorial">
+        <div class="tutorial-video">
+            <iframe class="tutorial-video__content" src="https://www.youtube.com/embed/-MT3adqmkio" frameborder="0"
```
Comment (jpowie01) [inline frontend/src/app/pages/tutorial-page/tutorial-page.component.html]:
```suggestion
            <iframe class="tutorial-video__content" src="https://www.youtube.com/embed/3WM0qtMMvdA" frameborder="0"
```

File: frontend/src/app/pages/tutorial-page/tutorial-page.component.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    &-info {
+        margin-top: 1em;
+        display: flex;
+        flex-direction: column;
+
+        &__next {
+            margin-top: 1em;
+
```
Comment (jpowie01) [inline frontend/src/app/pages/tutorial-page/tutorial-page.component.scss]: Unnecessary empty line.

File: frontend/src/app/pages/marker-tutorial-page/marker-tutorial-page.component.html [hunk 4]
```diff
@@ -1,108 +0,0 @@
-<app-nav-bar>
-</app-nav-bar>
-<mat-horizontal-stepper [linear]="true" #stepper>
-    <mat-step [stepControl]="formGroup">
-        <ng-template matStepLabel>Choose dataset</ng-template>
-        <section class="step-content">
-            <video loop #firstStepVideo>
-                <source src="assets/video/marker_step1.mp4" type="video/mp4">
```
Comment (jpowie01) [inline frontend/src/app/pages/marker-tutorial-page/marker-tutorial-page.component.html]: These assets can be safely removed! :)

File: frontend/src/app/pages/login-page/login-page.component.ts (hunk 1 above)
Comment (kolszewska) [reply frontend/src/app/pages/login-page/login-page.component.ts]: No.

**Pull request merged**